shell/browser/bluetooth/electron_bluetooth_delegate.cc

```diff
@@ -43,27 +72,31 @@ std::unique_ptr<content::BluetoothScanningPrompt>
 ElectronBluetoothDelegate::ShowBluetoothScanningPrompt(
     content::RenderFrameHost* frame,
     const content::BluetoothScanningPrompt::EventHandler& event_handler) {
+  LOG(INFO) << "In ElectronBluetoothDelegate::ShowBluetoothScanningPrompt";
```

Comment:
I think we can drop these debugging logs yeah?

Reply:
Resolved by https://github.com/electron/electron/pull/35416/commits/289d50cb87e88a4fc33c4e07d039d31866806f32